hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteCopyOnWrite.java

```diff
@@ -114,10 +116,28 @@ public void testCheckpointFails() throws Exception {
   }
 
   @Test
-  public void testSubtaskFails() throws Exception {
+  public void testSubtaskFailsWithEagerFailedWritesCleanPolicy() throws Exception {
+    testSubtaskFails()
+        // the last checkpoint instant was rolled back by subTaskFails(0, 2)
+        // with EAGER cleaning strategy
+        .assertNoEvent()
```

Comment:
Can we add new tests instead of modifying existing one?

Reply:
@danny0405, I only modified the name of test method and extracted part into a common method, not modify the existing one. PTAL.

Reply:
Just add a simple test for failed writes rollback, do not modify the existing tests.

Reply:
@danny0405, I have added a simple test for failed writes rollback. PTAL.